.github/workflows/release.yml

```
@@ -0,0 +1,101 @@
+name: "Release Pipeline"
+on:
+  push:
+    tags:
+      - v*.*.*
+
+  workflow_dispatch:
+
+jobs:
+  release-draft:
+    name: 'Draft release'
+    runs-on: ubuntu-latest
+    steps:
+        - name: "Create draft release"
+          uses: softprops/action-gh-release@v1
+          with:
+            draft: True
+
+  package-ubuntu:
+    needs: release-draft
+    name: 'Package Ubuntu ${{ matrix.UBUNTU }}'
+    runs-on: ubuntu-latest
+    container: 'ubuntu:${{ matrix.UBUNTU }}'
+    strategy:
+      fail-fast: true
+      matrix:
+        UBUNTU: ["18.04", "20.04", "20.10", "21.04"]
```

The 20.10 is not supported anymore and the 21.04 is going to reach End-Of-Life on January 20. This [table](https://en.wikipedia.org/wiki/Ubuntu_version_history#Table_of_versions) is very helpful.

General guideline:
- LTS releases (18.04, 20.04, 22.04, ...) are supported for five years
- point releases (20.10, 21.04, 21.10, 22.10, ...) are supported for nine months
- releases do overlap over time and we should support them if easy (e.g., even though the 21.10 is out, I am still on 21.04, as I still have enough time to upgrade)

```suggestion
        UBUNTU: ["18.04", "20.04", "21.04", "21.10"]
```